native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/email-native/reflect-config.json

```diff
@@ -0,0 +1,14 @@
+[
+{
+  "name":"com.sun.mail.handlers.text_plain",
```

Comment:
Can we format this file like this :
```json
[
    {
        "name": "com.sun.mail.handlers.text_plain",
        "methods": [
            {
                "name": "<init>",
                "parameterTypes": []
            }
        ]
    },
    {
        "name": "com.sun.mail.imap.IMAPStore",
        "methods": [
            {
                "name": "<init>",
                "parameterTypes": [
                    "javax.mail.Session",
                    "javax.mail.URLName"
                ]
            }
        ]
    },
    {
        "name": "com.sun.mail.smtp.SMTPTransport",
        "methods": [
            {
                "name": "<init>",
                "parameterTypes": [
                    "javax.mail.Session",
                    "javax.mail.URLName"
                ]
            }
        ]
    }
]
```

Reply:
Fixed with [f4700cc4b38824cd5bf4c1c3148247a3dc32a58a](https://github.com/ballerina-platform/module-ballerina-email/pull/561/commits/f4700cc4b38824cd5bf4c1c3148247a3dc32a58a)